File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/claim-management/shared/DTOs/credit-organization.dto.ts
export class CreditOrganization_DTO {
  public OrganizationId: number = 0;
  public OrganizationName: string = '';
  public IsActive: string = '';
  public CreatedOn: string = '';
  public CreatedBy: number = null;
  public ModifiedOn: string = '';
  public ModifiedBy: number = null;
  public IsDefault: boolean = false;
  public IsClaimManagementApplicable: boolean = false;
  public IsClaimCodeCompulsory: boolean = false;
  public IsClaimCodeAutoGenerate: boolean = false;
  public DisplayName: string = '';
  public SchemeApiIntegrationName: string = '';
}

File: Code/Websites/DanpheEMR/Services/ClaimManagement/DTOs/CreditOrganizationDTO.cs
using System;

namespace DanpheEMR.Services.ClaimManagement.DTOs
{
    public class CreditOrganizationDTO
    {
        public int OrganizationId { get; set; }
        public string OrganizationName { get; set; }
        public bool IsActive { get; set; }
        public DateTime CreatedOn { get; set; }
        public int CreatedBy { get; set; }
        public DateTime? ModifiedOn { get; set; }
        public int? ModifiedBy { get; set; }
        public bool IsDefault { get; set; }
        public bool IsClaimManagementApplicable { get; set; }
        public bool IsClaimCodeCompulsory { get; set; }
        public bool IsClaimCodeAutoGenerate { get; set; }
        public string DisplayName { get; set; }
    }
}
